sdk/monitor/monitor-opentelemetry-exporter/src/export/trace.ts

```diff
@@ -31,14 +29,14 @@ export class AzureMonitorTraceExporter implements SpanExporter {
 
   constructor(options: AzureExporterConfig = {}) {
     const connectionString = options.connectionString || process.env[ENV_CONNECTION_STRING];
-    this._logger = new ConsoleLogger(LogLevel.ERROR);
     this._options = {
       ...DEFAULT_EXPORTER_CONFIG
     };
+    diag.setLogLevel(this._options.logLevel);
```

Comment:
This should really only be set as part the application / startup, while not wrong it may override and consumers setting.